spec/integration/add_transactions_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe "Events" do
+  let(:application) { create(:application) }
+  let(:raw_event)   { create(:raw_event) }
+
+  before do
+    worker = ParseRawEventJob.new
+    worker.perform raw_event.id
+  end
+
+  it "include BEGIN COMMIT transaction" do
+    expect(Event.all.map(&:content)).to include "BEGIN COMMIT transaction"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.